### PR TITLE
Create ui-c8y-1021-13-1-label-for-display-as-function-overlapped-by-header-graft-release-cd.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [MTM-61821](https://cumulocity.atlassian.net/browse/MTM-61821)
Original PR: [7708](https://github.com/Cumulocity-IoT/cumulocity-ui/pull/7708)

[MTM-61821]: https://cumulocity.atlassian.net/browse/MTM-61821?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1021-13-1-label-for-display-as-function-overlapped-by-header-graft-release-cd.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: label for 'display-as' function overlapped by header [GRAFT][release/cd] (#7708)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1021-13-1-label-for-display-as-function-overlapped-by-header-graft-release-cd.md]: The Claude AI suggested to use the following title:
```suggestion
title: Fixed overlapping label for the "Display as" function in the header
```

File: content/change-logs/application-enablement/ui-c8y-1021-13-1-label-for-display-as-function-overlapped-by-header-graft-release-cd.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-61821
+version: 1021.13.1
+---
+label for 'display-as' function overlapped by header [GRAFT][release/cd] (#7708)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1021-13-1-label-for-display-as-function-overlapped-by-header-graft-release-cd.md]: The Claude AI suggested to use the following body:
```suggestion
In the Cumulocity user interface, the label for the "Display as" function in the header was overlapping with other header elements, making it difficult to read and select the desired option. This issue has now been resolved. The label for "Display as" is now properly positioned and no longer overlaps with the header, ensuring a clear and accessible user interface for all users. This change improves the usability and visual clarity of the "Display as" function in the Cumulocity header.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Review by janhommes — APPROVED** [review]

File: content/change-logs/application-enablement/ui-c8y-1021-13-1-label-for-display-as-function-overlapped-by-header-graft-release-cd.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-61821
+version: 1021.13.1
+---
+In the Cumulocity user interface, the label for the "Display as" function in the header was overlapping with other header elements, making it difficult to read and select the desired option. This issue has now been resolved. The label for "Display as" is now properly positioned and no longer overlaps with the header, ensuring a clear and accessible user interface for all users. This change improves the usability and visual clarity of the "Display as" function in the Cumulocity header.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1021-13-1-label-for-display-as-function-overlapped-by-header-graft-release-cd.md]:
```suggestion
In the {{< product-c8y-iot >}} user interface, the label for the "Display as" function in the header was overlapping with other header elements, making it difficult to read and select the desired option. This issue has now been resolved. The label for "Display as" is now properly positioned and no longer overlaps with the header, ensuring a clear and accessible user interface for all users. 
```

File: content/change-logs/application-enablement/ui-c8y-1021-13-1-label-for-display-as-function-overlapped-by-header-graft-release-cd.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Fixed overlapping label for the "Display as" function in the header
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1021-13-1-label-for-display-as-function-overlapped-by-header-graft-release-cd.md]:
```suggestion
title: Label for the "Display as" function no longer overlaps with other elements in the header
```

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**